nh_clinical/tests/user_allocation/helpers.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+def create_test_data(cls, cr, uid, iterations=3):
```

Comment:
Would this file be better off in something like nh_clinical/tests/common/helpers.py

That way other tests that might need this set up can use `from openerp.addons.nhclinical.tests.common import helpers`

Additionally Is it better to use the demo data for this setup?